### DISCUSSION TITLE
Openear

### DISCUSSION THREAD

**Discussion by lebleu666** [community/community#61746] [Discussions]
### Select Topic Area

Question

### Body

J'ai installé openear et ses encrypter le mode tetra et DMR comment décrypter le son merci

**Comment by queenofcorgis** [comment]
Hi @lebleu666 , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.

**Reply by lebleu666** [reply]
Hello, I would like to know how to configure DSDPLUS with my 2 openear and sdrtrunk applications.. I compress DSDPLUS and I cannot find how to make DMR encrypt itself decrypt... I need help.. I am ready to pay to configure the software with my 2 applications.. thank you

**Comment by lebleu666** [comment]
je cherche une personne pour installer DSDPLUS sur mes 2 application openear et sdr trunk suis de quebec beauport mon email est lecolbleu@msn .com merci
